scripts/reindex.py

```diff
@@ -80,6 +81,11 @@ def __init__(self, prog) -> None:
                     default=False,
                     action='store_true',
                     help='Index all matching bundles in the configured DSS instance.')
+parser.add_argument('--purge',
+                    default=False,
+                    action='store_true',
+                    help='Freeze the lambdas and purge the queues before reindexing. On success, lambdas are'
```

Comment:
```suggestion
                    help='Purge the queues before taking any actions on the index.')
```

The lambda enabling/disabling is an implementation detail of the purge action.